.github/workflows/state-pr.yaml

```
@@ -0,0 +1,17 @@
+name: "Close stale pull requests"
+on:
+  schedule:
+    - cron: "12 3 * * *" # arbitrary time not to DDOS GitHub
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days'
+        close-pr-message: 'Closed as inactive'
```

```suggestion
        close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
```

This might be a nice comment to add. I hope this repo doesn't have the problem but I have some nightmares of stalebot from this PR :P https://github.com/facebook/create-react-app/pull/8694#issuecomment-647027372